mir-webapp/src/main/solr/security.json

```diff
@@ -0,0 +1,42 @@
+{
+  "authentication": {
+    "blockUnknown": true,
+    "class": "solr.BasicAuthPlugin",
+    "credentials": {
+      "searcher": "0QWhQdOGUMTkjxqE0rPodHKa2gUEnRZnZ837YP/L4aw= 7A==",
+      "indexer": "0QWhQdOGUMTkjxqE0rPodHKa2gUEnRZnZ837YP/L4aw= 7A==",
+      "admin": "0QWhQdOGUMTkjxqE0rPodHKa2gUEnRZnZ837YP/L4aw= 7A=="
```

Comment:
What are these?

Reply:
`admin` and `alleswirdgut` which is okay for development and testing. Otherwise the solr-runner-maven plugin would need to generate these values and you would nee to configure them in the pom.

Reply:
Its also mentioned in the readme file, what these values are..